### ISSUE TITLE
TCP stats

### ISSUE THREAD

**Issue by gianarb** [gianarb/lb#26]
Open a TPC socket to manage stats and information like
- new host on
- new host out
- new frontend
- new call (for stats)

Case study:
manage stats on a TSDB like InfluxDB and do something when new host is in!

**Comment by liuggio** [comment]
:-1:  about this I think that maybe is better stay focused on the core things like balancing api etc..
I think they can be implemented in the second version... and maybe with an event system....

**Comment by gianarb** [comment]
Yes, also to remember that it could be a good feature because this tool is like the Tiburtina.. I hope to have a lot of possibile good metrics :)
